cmd/mattermost/commands/command.go

```diff
@@ -90,6 +90,18 @@ func createCommandCmdF(command *cobra.Command, args []string) error {
 		return errors.New("unable to find team '" + args[0] + "'")
 	}
 
+	// get the creator
+	creator, _ := command.Flags().GetString("creator")
+	user := getUserFromUserArg(a, creator)
+	if user == nil {
+		return errors.New("unable to find user '" + creator + "'")
+	}
+
+	// check if creator has permission to create slash commands
+	if !a.HasPermissionToTeam(user.Id, team.Id, model.PERMISSION_MANAGE_SLASH_COMMANDS) {
+		return errors.New("only team admins can create slash commands")
```

Comment:
Please can you change this text to `the creator must be a user who has permissions to manage slash commands`.

Reply:
done